test/StandardToken.js

```diff
@@ -5,8 +5,13 @@ var StandardTokenMock = artifacts.require('./helpers/StandardTokenMock.sol');
 
 contract('StandardToken', function(accounts) {
 
+  let token;
```

Comment:
It happened once to me because I was using the same 'testrpc' session for all tests, solved restarting it.